src/main/java/com/depromeet/tmp/util/EncryptionUtils.java

```diff
@@ -0,0 +1,31 @@
+package com.depromeet.tmp.util;
+
+import java.security.MessageDigest;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class EncryptionUtils {
+
+	public static String encryptSHA256(String s) {
+		return encrypt(s, "SHA-256");
+	}
+
+	private static String encrypt(String s, String messageDigest) {
+		try {
+			MessageDigest md = MessageDigest.getInstance(messageDigest);
+			byte[] passBytes = s.getBytes();
+			md.reset();
+			byte[] digested = md.digest(passBytes);
+			StringBuffer sb = new StringBuffer();
+			for(int i = 0; i < digested.length; ++i)
+				sb.append(Integer.toHexString(0xff & digested[i]));
+			return sb.toString();
+		}
+		catch (Exception e) {
+			log.error("EncryptionUtils Error Message : {}", e.toString());
```

Comment:
문자열 템플릿을 쓰기보다는 이렇게 해보시면 좋을 것 같네요.

```suggestion
			log.error("암호화 도중 예상하지 못한 예외가 발생했습니다", e);
```

- 문자열 템플릿 대신 예외 객체를 통째로 넘겨 stacktrace를 더 자세하게 찍히게 합니다. 그럼 디버깅 하기 쉬워지겠죠
- 에러가 발생했을 때 에러를 해결하려는 사람에게 상황 파악을 더 용이하게 해주기 위해 메세지를 자세하게 써줍니다.